### PR TITLE
fixes google.api_core.exceptions.InvalidArgument: 400 queries inside …

### PR DESCRIPTION
**Pull request by lygie2221**
…transactions must have ancestors Error when spidering

### CODE REVIEW

**Comment by marians** [comment]
Hi! Thanks for your contribution, and sorry for the late reply. I didn't see your PRs until now. Will have to update my notification settings.

So you are basically turning off transactions for job creation?

I have never run into the error you mention. Can you tell me how you are executing the script? What datastore version are you using with it? I went with "Firestore in datastore mode", but I recognize now that this is not documented.

**Comment by lygie2221** [comment]
> Can you tell me how you are executing the script?
I run
` docker run --rm -ti -v $PWD/secrets:/secrets quay.io/netzbegruenung/green-spider:latest --credentials-path secrets/datastore-writer.json jobs --url http://www.example.com`
and then
` make spider`
On a local copy of this repo.

> What datastore version are you using with it?
I am not shure how to get Datastore Version from google cloud

>So you are basically turning off transactions for job creation?
Yes, I don´t realy see the point of using transactions here. At this stage you only copy tasks from the datastore to a local dictionary. I don´t see how it needs to be atomic. Any exception that can happen here should be catchable and the task can continue with the valid data.

File: rating/contact_link.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ def rate(self):
             for link in self.check_results['hyperlinks'][url]['links']:
                 if link['text'].lower() == 'kontakt':
                     urls_with_contact_link += 1
+                    break
```
Comment (marians) [inline rating/contact_link.py]: As this change is unrelated to the other one, I suggest to remove it from the PR. I also changed that rater recently in a similar fashion.

**Comment by marians** [comment]
> I am not shure how to get Datastore Version from google cloud

I don't remember exactly, but at some point in my project I decided for a database type. Seems to me if the transaction support depends on that setting.

I'm fine with removing the transactions context. If it helps running with different database flavours, even better. And I agree it's not really needed here.

**Comment by marians** [comment]
Ich würde den PR gerne verwerfen.

1. Weil es so funktioniert, zumindest überall da, wo ich den Spider einsetze
2. Weil mir inzwischen wieder eingefallen ist, wozu diese Transaktion dienen sollte: Sie sollte sicher stellen, dass mehrere Worker sich nicht in die Quere kommen und beide den selben Job abrufen und löschen wollen.
